src/vs/code/electron-main/app.ts

```diff
@@ -166,7 +166,10 @@ export class CodeApplication extends Disposable {
 
 		const isUrlFromWebview = (requestingUrl: string | undefined) => requestingUrl?.startsWith(`${Schemas.vscodeWebview}://`);
 
+		const allowedPermissions = new Set(['pointerLock']);
```

Comment:
Does this need to be enabled for the root document too? The only use case seems to be for webviews

Reply:
Yes the permission did not work until I added it to the root. That is to say I also needed the permission on the containing frame in order for the child to function in my extension's Webview.